composer.json

```diff
@@ -3,14 +3,14 @@
   "description": "This class displays structured information about one or more expressions that includes its type and value. Arrays and objects are explored recursively with values indented to show structure.",
   "type": "library",
   "license": "GPL-3.0-or-later",
-  "minimum-stability": "dev",
+  "minimum-stability": "stable",
```

Comment:
Let the package be stable release, and nobody can guarantee that developers want to install unstable version.